### ISSUE TITLE
Validate benchmarks

### ISSUE THREAD

**Issue by skjolber** [skjolber/csv-benchmark#1]
The benchmarks are showing very good results, it would be interesting to get some more eyes on and also running them on more hardware.

**Comment by aroger-r7** [comment]
hey just came across your benchmark, had a quick look there is quite a lot.
first thing is that you might want to upgrade sfm to version 6.0.9.

**Comment by skjolber** [comment]
@aroger-r7  Right, bumped, should the code be adjusted as well as the dependency?

**Comment by aroger-r7** [comment]
there CsvReader has moved package but I think that should be it

**Comment by arnaudroger** [comment]
so run it locally with better -i 5 -wi 5 and f 5 - took more than 24 hours. but you mapper is faster. I wonder if that comes down to the number parsing. unfortunately, the gradle plugin does not build a shaded jar. but I'll try to get some profiling info to see.

**Comment by skjolber** [comment]
Uh yeah, the run time should be adjusted, kind of long now.

Edit: :+1: I read that a bit too fast it seems

**Comment by skjolber** [comment]
There is actually two simpleflatmapper dependencies, the other is in https://github.com/skjolber/csv-benchmark/blob/master/jmh-benchmark/build.gradle and I forgot to bump it the first time around. Edit: So that might have skewed the results. Have not had the time to run new ones yet.

**Comment by skjolber** [comment]
@arnaudroger I see the number parsing has been improved in sfm - did that affect the benchmarking results?

**Comment by arnaudroger** [comment]
slightly, but your mapper still faster. got a bit of improvement by doing a
parseInt on char[] directly but still quite far :)

**Comment by arnaudroger** [comment]
just pushed 6.0.10 with some pref improvement for sfm-csv. still not as fast :) not that surprising. I don't think I can get it as low because the support for constructor injection, sub object and other yaml comment.

**Comment by skjolber** [comment]
I'll refresh the benchmark results anyhow (within 2-3 weeks).

**Comment by skjolber** [comment]
@arnaudroger Benchmarks refreshed. My gut feeling is that SFM improved somewhat.
